Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs

```diff
@@ -130,20 +130,7 @@ private static async Task ConstructTextureAsync(this GltfObject gltfObject, Gltf
                     if (texture == null)
                     {
                         Debug.LogWarning($"Attempting to load asset at {path}");
-
-                        using (FileStream stream = File.Open(path, FileMode.Open))
-                        {
-                            imageData = new byte[stream.Length];
-
-                            if (gltfObject.LoadAsynchronously)
-                            {
-                                await stream.ReadAsync(imageData, 0, (int)stream.Length);
-                            }
-                            else
-                            {
-                                stream.Read(imageData, 0, (int)stream.Length);
-                            }
-                        }
+                        imageData = File.ReadAllBytes(path);
```

Comment:
Why remove the async read?

Reply:
It doesn't work when built in visual studio. You get an access denied when attempting to open the stream. We could duplicate this to a temp file but it seemed higher priority to get this code in with follow up load async improvements in a different issue.

Reply:
Specifically an UnauthorizedAccessException

Reply:
Please revert this. I've got a proper fix